src/vsr/replica.zig

```diff
@@ -1205,21 +1285,21 @@ pub fn ReplicaType(
         /// ⟨start_view v, l, n, k⟩ messages to the other replicas, where l is the new log, n is the
         /// op number, and k is the commit number.
         fn on_do_view_change(self: *Self, message: *Message) void {
```

Comment:
Update the comment that DVC serves two purposes: 

* for primary, to become part of DVC quorum
* for backup, to force view_jump.